sdk/python/jobs/pipelines/2e_image_classification_keras_minist_convnet/image_classification_keras_minist_convnet.ipynb

```diff
@@ -108,22 +108,52 @@
    "metadata": {},
    "source": [
     "## 1.4 Prepare Job Input\n",
-    "By defining `Input`, you create a reference to the data source location. The data remains in its existing location, so no extra storage cost is incurred."
+    "By defining `Input`, you create a reference to the data source location. We will use local locations in this example.\n",
+    "\n",
+    "In this sample, we used [`mnist`](https://learn.microsoft.com/en-us/azure/open-datasets/dataset-mnist?tabs=azure-storage) as the target task. You can also try [`fashion-mnist`](https://github.com/zalandoresearch/fashion-mnist?tab=readme-ov-file#get-the-data), which has the same structure as `mnist` but of a harder task.\n",
+    "\n",
+    "Note that, **you may meet failure in the below block if you're in a private network and have no access to the public blob**. If so, you can try downloading from [official site of `fashion-mnist`](https://github.com/zalandoresearch/fashion-mnist?tab=readme-ov-file#get-the-data) or another machine, and unzip the downloaded file to [mnist](./mnist/)."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "name": "define-input"
```

Comment:
@elliotzh azure-docs-pr is failing due to this update: https://github.com/MicrosoftDocs/azure-docs-pr/pulls Could you add it back?